cmd/kubeadm/app/phases/kubelet/kubelet.go

```diff
@@ -81,7 +92,15 @@ func ConsumeBaseKubeletConfiguration(nodeName string) error {
 		return err
 	}
 
-	kubeletCfg, err := client.CoreV1().ConfigMaps(metav1.NamespaceSystem).Get(kubeadmconstants.KubeletBaseConfigurationConfigMap, metav1.GetOptions{})
+	var majorMinorVersion string
+	kubeletVersion, err := preflight.GetKubeletVersion(utilsexec.New())
+	if err != nil {
+		majorMinorVersion = "1.9"
```

Comment:
just fallback on kubeadm's own version here instead in a generic sense or something?
you can use `pkg/version` for that